### DISCUSSION TITLE
Seeking Beta Testers for Linux and MacOS

### DISCUSSION THREAD

**Discussion by thelastoutpostworkshop** [thelastoutpostworkshop/arduino-maker-workshop#2] [Announcements]
To ensure the extension works seamlessly across all platforms, I am inviting developers and makers knowledgeable in Linux and MacOS to help test its functionality.

Your feedback on features such as board management, compilation, upload, and overall usability is invaluable in improving the extension.

What I am looking for:

Linux and MacOS users experienced with Arduino development.
Ability to test features like serial monitor integration, CLI configuration, and compatibility with different boards.
If you’re interested in contributing to the development of this extension and providing feedback, please let me know. Your expertise is greatly appreciated!

**Comment by schack-lindemann** [comment]
I would like to enlist as beta-tester for the macOS version if needed. I'm not a macOS developer but mainly program for microcontrollers.

**Reply by thelastoutpostworkshop** [reply]
That's perfect, I am not sure when I will start, but I will let you know

**Reply by thelastoutpostworkshop** [reply]
Are you on Mac OS or Mac OS ARM ?

**Reply by thelastoutpostworkshop** [reply]
A version of the extension for Mac OS is now available, you can test it.

**Reply by nomiknomik** [reply]
Hi, I would like to test it too on my Mac Intel

**Reply by schack-lindemann** [reply]
I just discovered this new Mac version. Cool! Will be testing it over the next weeks. I'm running it on macOS 15.2 on ARM.

**Comment by toolpathguy** [comment]
I have an nVidia Jetson AGX Dev kit showing soon that will be running Linux. I would like to sign up for the linux testing. Same thing for me, I'm mainly programming mircocontrollers.

**Comment by pixelEDI** [comment]
Working great on Arch Linux (6.12.9-arch1-1), Sway, AMD GPU & CPU, visual-studio-code-bin from AUR Version 1.96.2

- [x] Arduino R4 Wi-Fi
- [x] ESP32-C3 XIAO
- [x] Serial Monitor
- [x] Adafruit_SSD1306 library installed

Nice project!

**Reply by thelastoutpostworkshop** [reply]
Thank a lot for taking the time to report the results!

**Comment by fredodupoux** [comment]
I work on both MacOS Apple Silicon and Fedora Linux. If you'd like feedback I can beta test for you.
Thanks.

**Comment by KevynWithaY** [comment]
Do you think you could publish this extension to the Open VSX extension marketplace, please? I use Windsurf IDE which is a fork of VS Code, and Microsoft won't allow forks to use the official extension gallery, or support installs from that site. https://github.com/eclipse/openvsx/wiki/Publishing-Extensions

**Reply by thelastoutpostworkshop** [reply]
I would like to do that, but the Arduino Maker Workshop extension needs the Serial Monitor extension, which is not available in openvsx marketplace and the C/C++ IntelliSense extension which is 2 years old on the openvsx marketplace.
